docs/tutorials/reuse-ssh-connections.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Reuse SSH Connections
```

Comment:
Since there is the possibility of adding a Windows section to this tutorial, I would rather keep the OS out of the title/filename. And actually, this will already work on Windows as-is if you are using, for example, OpenSSH in Cygwin.